spring-boot-project/spring-boot-docs/src/docs/asciidoc/messaging/jms.adoc

```diff
@@ -8,13 +8,13 @@ Spring Boot also auto-configures the necessary infrastructure to send and receiv
 
 
 [[messaging.jms.activemq]]
-=== ActiveMQ "Classic" Support
```

Comment:
We can certainly remove this quotes, but this should be done separately as it's unrelated to reinstating support for auto-configuring an embedded broker.